castai/add-castai-removal-disabled/artifacthub-pkg.yml

```diff
@@ -0,0 +1,23 @@
+name: add-castai-removal-disabled
+version: 1.0.0
+displayName: Add CAST AI removal disabled
+createdAt: "2023-04-10T20:11:12.000Z"
+description: >-
+  CAST AI will not downscale a node that includes a pod with the autoscaling.cast.ai/removal-disabled="true" label on it, this protects sensitive workloads from being evicted and can be attributed to any pod to protect against unwanted downscaling. This policy will mutate jobs and cronjobs to add the removal-disabled label to protect against eviction. 
```

Comment:
Changes made to description in the policy should be reflected here as well.

Reply:
I updated the policy to be labels instead of annotations so I think the description is consistent now.